docs/developers/contributing/testing.md

```diff
@@ -33,28 +33,53 @@ from unit tests to integration and functional tests.
 All of `napari` tests are located in folders named `_tests`. We keep our unit
 tests located in the individual folders with the modules or functions they are
 testing (e.g. the tests for the `Image` layer are located in
-`/napari/layers/image/_tests` alongside the `Image` layer code). Our integration and
-functional tests are located in
-the `napari/_tests` folder at the top of the repository.
-
-We also strive to unit test as much of our model file and utils code independently of
-our GUI code. These tests are located in the `napari/layers`, `napari/components`,
-and `napari/utils` folders. Our GUI code is tests in the `napari/_tests`,
-`napari/_qt`, `napari/_vispy` folders. The tests in these three folders are ignored
-when we run them in the subset of our
+[`/napari/layers/image/_tests`](https://github.com/napari/napari/tree/main/napari/layers/image/_tests)
+alongside the `Image` layer code).
+Our integration and functional tests are located in
+the [`napari/_tests`](https://github.com/napari/napari/tree/main/napari/_tests)
+folder at the top of the repository.
+
+We also strive to unit test as much of our model and utils code independently of
+our GUI code. These tests are located in the following folders:
+
+* [`napari/layers`](https://github.com/napari/napari/tree/main/napari/layers)
+* [`napari/components`](https://github.com/napari/napari/tree/main/napari/components)
+* [`napari/utils`](https://github.com/napari/napari/tree/main/napari/utils)
+
+Our GUI code is tests in the following folders:
+
+* [`napari/_tests`](https://github.com/napari/napari/tree/main/napari/_tests)
+* [`napari/_qt`](https://github.com/napari/napari/tree/main/napari/_qt)
+* [`napari/_vispy`](https://github.com/napari/napari/tree/main/napari/_vispy)
+
+The tests in these three folders are ignored when we run them in the subset of our
 [continuous integration](https://en.wikipedia.org/wiki/Continuous_integration)
 workflows that run in a headless environment (without a Qt backend).
 When we are testing "non-GUI" code in a way that requires a GUI backend, they are
-placed here. The `napari/plugins` folder contains a mix of tests.
+placed here.
+
+The `napari/plugins` folder contains tests related to plugins.
 
-Pytest fixtures that are available globally to aid testing live in `napari/conftest.py`.
+Pytest fixtures to aid testings live in:
+
+* [`napari/conftest.py`](https://github.com/napari/napari/blob/main/napari/conftest.py)
+* [`napari_builtins/_tests/conftest.py`](https://github.com/napari/napari/blob/main/napari_builtins/_tests/conftest.py)
```

Comment:
I would put this separately and mention that it's fixtures for the `builtins` plugin

Reply:
Will do, but I think we need a section describing what the builtins plugin is. see: https://github.com/napari/docs/issues/321